src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {
+    void process(List<FruitTransaction> fruitTransactions);
+
+    Map<String, Integer> getStorage();
```

Comment:
this is the redundant method. Your ShopService shouldn't be an entry point for storage retrieval. Storage is an independent entity and shouldn't be associated with ShopService